src/software/jetson_nano/services/service.h

```diff
@@ -1,17 +0,0 @@
-#pragma once
-/**
- * A service defines an interface with hardware/software on the robot.
- */
-class Service
-{
-   public:
-    /**
-     * Start the service.
-     */
-    virtual void start() = 0;
```

Comment:
whats the problem with the start/stop idea?

Reply:
My goal was to have services respect the start/stop interface cleanly so that when the emergency stop was pressed/released we can have hooks to transition in/out of these services

Turns out that we don't need it, we only have 3 services, out of which the network service needs to always be running. The motor service also can't "stop" because of the heartbeat feature, it needs to still poll, just with an e-stop primitive.